CONTRIBUTING.md

```diff
@@ -132,25 +140,25 @@ project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Commit your changes in logical chunks. Please adhere to these [git commit
-   message guidelines](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
-   or your code is unlikely be merged into the main project. Use Git's
-   [interactive rebase](https://help.github.com/articles/interactive-rebase)
-   feature to tidy up your commits before making them public.
+4. Please commit your changes in logical chunks, and try to provide clear commit
```

Comment:
I wonder if we need this sentence. Most beginner PRs will be squash-and-merged. It's easier for people to add a lot of increment commits without having to worry about keeping it tidy. Then it's all flattened into a single commit in the end. This is what we do as well.

What do you think?

Reply:
I think it does not hurt the review process if the commits are neat and tidy; I tend to go through a PR commit-by-commit to understand how the contribution was built up from the ground up, although sometimes I simply give up if the dead ends are also there.